functions/functions/transfer-chat.js

```diff
@@ -6,10 +6,12 @@ exports.handler = JWEValidator(async function (context, event, callback) {
 
 	// setup a response object
 	const response = new Twilio.Response();
-	response.appendHeader('Access-Control-Allow-Origin', '*');
+	response.appendHeader('Access-Control-Allow-Origin', 'http://localhost');
+	response.appendHeader('Access-Control-Allow-Origin', 'https://flex.twilio.com');
```

Comment:
Wouldn't this break the plugin for customers who self host on a custom domain?

Reply:
Yes, good point! (I'm a customer that uses only the hosted instance, so didn't think of that) I guess the original Access-Control-Allow-Origin: * is the way to go here :) I'll change it back :)